kroxylicious-app/src/assembly/kroxylicious-start.sh

```diff
@@ -64,9 +64,10 @@ NETTY_NATIVE_LIB=$(native_library_path netty)
 LZ4_NATIVE_LIB=$(native_library_path lz4-java/net/jpountz/util)
 SNAPPY_NATIVE_LIB=$(native_library_path snappy/org/xerial/snappy/native)
 ZSTD_NATIVE_LIB=$(native_library_path zstd-jni)
+ZSTD_FULLY_QUALIFIED=$(ls "${ZSTD_NATIVE_LIB}"/libzstd-jni-*.so)
 NATIVE_LIB_PATH="${NATIVE_LIB_PATH:-${NETTY_NATIVE_LIB}:${LZ4_NATIVE_LIB}:${SNAPPY_NATIVE_LIB}:${ZSTD_NATIVE_LIB}:${LD_LIBRARY_PATH:-}}"
-echo "setting java.library.path=${NATIVE_LIB_PATH}"
```

Comment:
the full command line gets echo'd anyway so this is effectively duplicate logging and not adding any value.